pkg/upstream/write.go

```diff
@@ -198,3 +232,557 @@ func maybeEncryptIdentityConfig(identityConfig *kotsv1beta1.IdentityConfig) ([]b
 
 	return b.Bytes(), nil
 }
+
+// FindReplicatedSDKChart will look for a chart with the name "replicated" or "replicated-sdk" in the archive
```

Comment:
should update this comment to reflect that it uses `replicatedChartNames` and not hardcoded values
```suggestion
// FindReplicatedSDKChart will look for a chart with the name "replicated" or "replicated-sdk" in the archive
```